dashflux/src/app/components/things/add-thing-dialog/add-thing-dialog.component.html

```diff
@@ -1,24 +1,24 @@
-<form [formGroup]="addClientForm" (ngSubmit)="onAddClient()">
-  <h1 mat-dialog-title>Add client</h1>
+<form [formGroup]="addThingForm" (ngSubmit)="onAddThing()">
+  <h1 mat-dialog-title>Add thing</h1>
   <div mat-dialog-content fxLayout="column">
     <input type="hidden" formControlName="id">
-    <mat-form-field>
-      <mat-select placeholder="Client type" formControlName="type">
+    <mat-form-field *ngIf="addThingForm.get('id').value === null">
```

Comment:
Please add comment for this condition, or make bool variable with meaningful name for it.